SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+## Supported Versions
+
+We are currently providing security updates to the following versions: 
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.21.x   | :white_check_mark: |
+| 0.20.x   | :white_check_mark: |
+| 0.19.x   | :x: |
+| 0.18.x   | :x: |
+| < 0.18   | :x: |
+
+## Reporting a Vulnerability
+
+We will use [keybase](https://keybase.io) as the vehicle for reporting security issues as that gives us a
```

Comment:
Why, I genuinely ask, do you prefer to use Keybase over just opening a new Github issue?

Reply:
Discussing security severe security problems in the open allows attackers to see that as well. Once there is a security patch there is usually a race between threat actors to exploit and users to upgrade. We want users to be able to upgrade before they become attacked.

Reply:
AFAIK there is no way to limit a github issue to just be visible to the reporter and the maintainers. That would be ideal. Using the tools that we already know and use day to day.

Reply:
Thanks, @hamnis, for the explanation.

Reply:
GitHub's Security feature gives us a private fork, which is helpful for sharing code and discussing the solution among project owners and invited parties.  I'm unclear whether the security advisory exposes draft-status comments to the public.  We're learning about this tool on the fly, but I think Keybase is useful at minimum for initial disclosure.